wire-ios/Wire-iOS UnitTests/DeviceInfoViewModelTests.swift

```diff
@@ -0,0 +1,130 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import XCTest
+@testable import Wire
+
+final class DeviceInfoViewModelTests: XCTestCase {
+    let mockDeviceActionsHandler = MockDeviceDetailsViewActions()
+    let mockConversationUserClientDetailsActions = MockConversationUserClientDetailsActions()
+    var deviceInfoViewModel: DeviceInfoViewModel!
+
+    override func setUp() {
+        super.setUp()
+        self.deviceInfoViewModel = DeviceInfoViewModel(certificate: .mockExpired,
+                                                       title: "",
+                                                       addedDate: "",
+                                                       proteusID: "",
+                                                       mlsThumbprint: "",
+                                                       isProteusVerificationEnabled: true,
+                                                       userClient: MockUserClient(),
+                                                       isSelfClient: false,
+                                                       gracePeriod: 0,
+                                                       isFromConversation: false,
+                                                       actionsHandler: mockDeviceActionsHandler,
+                                                       conversationClientDetailsActions: mockConversationUserClientDetailsActions,
+                                                       showDebugMenu: false)
```

Comment:
```suggestion
        self.deviceInfoViewModel = DeviceInfoViewModel(
            certificate: .mockExpired,
            title: "",
            addedDate: "",
            proteusID: "",
            mlsThumbprint: "",
            isProteusVerificationEnabled: true,
            userClient: MockUserClient(),
            isSelfClient: false,
            gracePeriod: 0,
            isFromConversation: false,
            actionsHandler: mockDeviceActionsHandler,
            conversationClientDetailsActions: mockConversationUserClientDetailsActions,
            showDebugMenu: false
        )
```

Reply:
Could be marked as nitpick. And be approved

Reply:
I didn't approve yet because of my other open conversations